pkg/actions/nodegroup/get.go

```diff
@@ -78,11 +80,25 @@ func (m *Manager) GetAll() ([]*manager.NodeGroupSummary, error) {
 	return summaries, nil
 }
 
-func getInstanceTypes(ng *awseks.Nodegroup) string {
+func (m *Manager) getInstanceTypes(ng *awseks.Nodegroup) string {
 	if len(ng.InstanceTypes) > 0 {
 		return strings.Join(aws.StringValueSlice(ng.InstanceTypes), ",")
 	}
-	logger.Info("no instance types reported by EKS for nodegroup %q", *ng.NodegroupName)
+
+	if ng.LaunchTemplate != nil {
```

Comment:
this could be refactored to keep the happy path on the left and exit early, e.g.:
```
if ng.LaunchTemplate == nil {
  logger.Info("no instance type found for nodegroup %q", *ng.NodegroupName)
  return "-"
}
etc
```
this works too but it'll be clearer that way :)
wdyt?

Reply:
done!